redash/query_runner/cass.py

```diff
@@ -14,7 +14,7 @@
     enabled = False
 
 
-class Cassandra(BaseQueryRunner):
+class Cassandra(BaseSQLQueryRunner):
```

Comment:
Correct, because you need to implement `get_schema` instead (`get_tables` is a method BaseSQLQueryRunner uses for `get_schema`).